perf/terraform/configs/README.md

```diff
@@ -0,0 +1,13 @@
+# Configs
+
+The terraform configs defined in this directory are used to provision the infrastructure for the libp2p perf tests.
+
+The configs are named after the type of backend they use. The defaults for what parts of infrastructure they provision differ between the two.
+
+## local
+
+Terraform state in this configuration will be stored locally. The defaults are configured for local development i.e. terraform apply will bring up ALL the infrastructure needed to run the perf tests. It will skip the steps required only for the CI environment.
```

Comment:
```suggestion
Terraform state in this configuration will be stored locally. The defaults are configured for a single performance benchmark run, i.e. `terraform apply` will bring up short-lived infrastructure only. It will skip long-lived infrastructure like the clean-up Lambda and the instance launch template.
```